src/wmtk/MultiMeshManager.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <tuple>
+#include "attribute/AttributeManager.hpp"
+#include "attribute/AttributeScopeHandle.hpp"
+#include "attribute/MeshAttributes.hpp"
+#include "Tuple.hpp"
+#include <spdlog/spdlog.h>
+
+namespace wmtk
+{
+class Mesh;
+class MultiMeshManager
+{
+    public:
+
+    MultiMeshManager();
+    ~MultiMeshManager();
+    MultiMeshManager(const MultiMeshManager& o);
+    MultiMeshManager(MultiMeshManager&& o);
+    MultiMeshManager& operator=(const MultiMeshManager& o);
+    MultiMeshManager& operator=(MultiMeshManager&& o);
+
+    //=========================================================
+    // Storage of MultiMesh
+    //=========================================================
+
+    bool is_parent_mesh() const { return m_is_parent_mesh; }
```

Comment:
if MeshAttributeHandle is aware of whether it's in a `null` state then this bit is redundant.